LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentApi.kt

```diff
@@ -0,0 +1,88 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.core.BraintreeClient
+import com.braintreepayments.api.localpayment.LocalPaymentNonce.Companion.fromJSON
+import org.json.JSONException
+import org.json.JSONObject
+
+internal class LocalPaymentApi(private val braintreeClient: BraintreeClient) {
+    fun createPaymentMethod(
+        request: LocalPaymentRequest,
+        callback: LocalPaymentInternalAuthRequestCallback
+    ) {
+        val returnUrl =
+            braintreeClient.getReturnUrlScheme() + "://" + LocalPaymentClient.LOCAL_PAYMENT_SUCCESS
+        val cancel =
+            braintreeClient.getReturnUrlScheme() + "://" + LocalPaymentClient.LOCAL_PAYMENT_CANCEL
```

Comment:
Can we use the Kotlin $ string interpolation here?

Reply:
+1. The Kotlin way can help to improve readability.